src/main/res/values/strings.xml

```diff
@@ -52,7 +52,7 @@
     <string name="mute">Mute</string>
     <string name="muted">Muted</string>
     <string name="ephemeral_messages">Disappearing Messages</string>
-    <string name="ephemeral_messages_hint">Applies to all members of this chat if they use Delta Chat; they can still copy, save, and forward messages or use other e-mail clients.</string>
+    <string name="ephemeral_messages_hint">Applies to all members of this chat if they use Delta Chat; they can still copy, save, and forward messages or use other apps.</string>
```

Comment:
I think that "if they use Delta Chat" and "or use other apps" may be unclear for users who are not used to being able to use different apps to participate in the same chat. Not sure what a better wording could be, but we could remove one of them:

```suggestion
    <string name="ephemeral_messages_hint">Applies to all members of this chat if they use Delta Chat; they can still copy, save, and forward messages.</string>
```

Reply:
yeah, i think, "copy, save, and forward" is good enough to make clear the option is not 100% and needs trust

also that is shorter, so chances are higher ppl read :)